### ISSUE TITLE
Can't enable TSL/SSL Authentication on Poste.io

### ISSUE THREAD

**Issue by StefanoVe** [caprover/one-click-apps#1072]
I've installed Poste.io through one-click-apps and followed @locvfx's comment in #231 to have both the web interface and the mail server itself authenticated with SSL but poste doesn't seem to recognise the copied over certificates, resulting in a "connection refused" error on the LE row in connection diagnostic.

I've setup `mail.<domain>` as a mailserver hostname and my DNS is configured as following:

```

   ...subdomains
 /
<domain> ---- MX ---->  mail.<domain> --- TXT --->"v=spf1 a mx ip4:<server ip> ~all"
       \                             \
        A ----> <server ip>           \
                                        A ---> <server ip>
   
                                     
```

Since my poste installation runs under `poste.vps.<domain>` I've also tried to add `mail.<domain>` in the caprover's HTTP settings, enabling SSL through there and then copying those certificates to poste's ssl folder but with no luck.

Everything else works but this so I'm really not sure what could be the problem.

Edit: ended up being an error on my part as I was missing the PTR record to my domain in my VPS, the LE challenge still results in an error but both Spark and Gmail now connect without any issue to the email server

**Comment by coffseducation** [comment]
This affects me also.
I added into configs the 3 additions of; Path in App - path on host details 
but re: Caprover will tell you to change nginx config : DONT FOLLOW
how does one revert these changes? I probably did follow the instructions!

ps emails are sending for me in my desktop client, Kmail, but I cannot send invoices from Invoice Ninja and this seems to be the problem.

Let me know if you find a solution!

**Comment by StefanoVe** [comment]
To revert the nginx changes though you can simply delete everything in that textarea, caprover will then re-apply the default configuration. 

Still haven't found a solution to enable TLS

**Comment by coffseducation** [comment]
Apologies I failed to respond. Thanks for the info.
Closed? Did you get this sorted or give up?
I tried all of the advice in combinations but still could not get it working.

**Comment by coffseducation** [comment]
Posteo.io has these settings
![image](https://github.com/caprover/one-click-apps/assets/99161534/aea45ecb-69e2-4075-a929-b6c0300dc296)
I copied the keys from the directory:
/captain/data/letencrypt/etc/live/YOUR-APP-NAME/privkey.pem etc
I then pasted them into a new file and uploaded.
I not sure what file goes to each upload so I tried a mix. 
Still not working.

How is iRedmail? It looks to set ssl also, so I assume the same issue.
I suppose Caprover is really not setup to host an email server. It would seem be to go with a dedicated hosting service.
